maltd-frontend/cypress/support/pageObjects/user-page.js

```diff
@@ -0,0 +1,7 @@
+/* global cy */
+class UserPage {
+  static getBackButton() {
+    return cy.get(".size-large");
```

Comment:
feel free to add an `id` in the jsx code to any element you need to find in the tests - i think it is a cleaner approach than looking for a classname

Reply:
Thank you. That would be awesome. Cypress generally uses class names to identify locators. Id's are super helpful. Will add them as needed.